e2e/test/common.bats

```diff
@@ -146,6 +148,7 @@ function ts_project() {
         --source-map) shift; source_map="source_map = True," ;;
         --declaration) shift; declaration="declaration = True," ;;
         --composite) shift; composite="composite = True," ;;
+        --mockTranspiler) shift; composite="transpiler = mock," ;;
```

Comment:
```suggestion
        --mockTranspiler) shift; transpiler="transpiler = mock," ;;
```

And then `$transpiler` should be inserted in the appropriate place in the heredoc string below.